assets/js/modules/reader-revenue-manager/components/setup/SetupMain.stories.js

```diff
@@ -27,8 +32,27 @@ function Template() {
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
+Default.scenario = {
+	label: 'Modules/ReaderRevenueManager/Setup/SetupMain/Default',
+	delay: 250,
+};
 
 export default {
 	title: 'Modules/ReaderRevenueManager/Setup/SetupMain',
 	component: SetupMain,
+	decorators: [
+		( Story ) => {
+			const registry = createTestRegistry();
+
+			registry
+				.dispatch( MODULES_READER_REVENUE_MANAGER )
+				.receiveGetPublications( [] );
+
+			return (
+				<WithTestRegistry registry={ registry }>
```

Comment:
Please use `WithRegistrySetup` and the setup `func` instead – we should almost never need `WithTestRegistry`

Reply:
Thanks for the reminder! I had forgotten this, and mostly copied the decorator from one of the existing stories which use `WithTestRegistry`, e.g.:

https://github.com/google/site-kit-wp/blob/7705f0ce9e66f5325ef203e73a6f7a44131a4722/assets/js/components/Header.stories.js#L286-L300

Anyhow - good to be reminded, I've updated the story!